docs/REST API/rest-api.md

```diff
@@ -6376,32 +6377,72 @@ None.
 }
 ```
 
-### Get Apple Business Manager (ABM)
+### List Apple Business Manager (ABM) tokens
 
 _Available in Fleet Premium_
 
-`GET /api/v1/fleet/abm`
+`GET /api/v1/fleet/abm/tokens`
```

Comment:
Maintain support for the old `GET /abm` endpoint for backwards compatibility.

If there are multiple Apple Business Manager connections `GET /abm` returns the following error: 

"This API endpoint has been deprecated in favor of the new GET /abm/tokens API endpoint. Please use the new endpoint documented here: https://fleetdm.com/learn-more-about/apple-business-manager-api"

Reply:
Redirect for the URL in the error message is here: https://github.com/fleetdm/fleet/pull/21043/files#diff-0c6120927d3e65309562b5b15b261d3298d05fcb0ff18e5b6512ee683b7ad6b3R532